src/main/kotlin/gg/flyte/christmas/minigame/games/MusicalMinecarts.kt

```diff
@@ -237,16 +237,16 @@ class MusicalMinecarts : EventMiniGame(GameConfig.MUSICAL_MINECARTS) {
         }
         super.eliminate(player, reason)
 
-        var value = "$roundNumber round${if (roundNumber > 1) "s" else ""}"
+        val value = "$roundNumber ʀᴏᴜɴᴅ${if (roundNumber > 1) "ѕ" else ""}"
```

Comment:
Is this 's' here in small text? Honestly can't tell.

Reply:
You can see it in the diff